crates/holochain/tests/tests/regression/mod.rs

```diff
@@ -4,6 +4,11 @@ use holochain::conductor::conductor::WASM_CACHE;
 use holochain::{sweettest::*, test_utils::retry_fn_until_timeout};
 use holochain_wasm_test_utils::TestWasm;
 
+mod dht_location;
```

Comment:
Moved these to the top of the file (hope that's okay) and added one new mod.

Reply:
Yes from me, I prefer this ordering. Having a roughly standard layout for files makes it so much easier to navigate the code.

Reply:
I've seen the `tests` mod listed at the bottom of files, but I don't see the point. If we keep mod doc comments inside the module file, all declarations can be moved to the top.